src/main/java/TaskManager.java

```diff
@@ -0,0 +1,103 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+    ArrayList<Task> tasks;
+
+    public TaskManager() {
+        this.tasks = new ArrayList<>();
+    }
+
+    protected void printWithSeparator(String... strings) {
+        System.out.println("\t____________________________________________________________");
+        for (String s : strings) {
+            System.out.println("\t" + s);
+        }
+        System.out.println("\t____________________________________________________________");
+    }
+
+    public void welcomeUser() {
+        printWithSeparator("Hello from JRobo! I'm your personal assistant!",
+                "Nice to meet you. What can I do for you?");
+    }
+
+    public void displayTaskList() {
+        if (tasks.size() == 0) {
+            printWithSeparator("You have no tasks to list.");
+            return;
+        }
+        System.out.println("\t____________________________________________________________");
+        System.out.println("\tHere are the tasks in your list:");
+        int taskCount = Task.getTaskCount();
+        for (int i = 0; i < taskCount; i++) {
+            Task t = tasks.get(i);
+            System.out.println("\t" + (i + 1) + "." + t);
+        }
+        System.out.println("\t____________________________________________________________");
+    }
+
+    public void markTask(String input) {
+        String numberString = input.trim();
+        try {
+            int taskNumber = Integer.parseInt(numberString);
+            Task task = tasks.get(taskNumber - 1);
+            if (task.getStatusIcon().equals("X")) {
+                printWithSeparator("This task is already marked");
+                return;
+            }
+            task.markAsDone();
+            printWithSeparator("Nice! I've marked this task as done:", "\t[" + task.getStatusIcon() + "] "
+                    + task.getDescription());
```

Comment:
Good use of line wrapping here so that code is kept within 120 characters per line.
Correct indentation of wrapped line too at 8 spaces.